package.json

```diff
@@ -7,6 +7,7 @@
     "@fontsource/roboto": "^4.5.0",
     "@material-ui/core": "^4.12.2",
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.60",
```

Comment:
Do we also want to commit the new addition to `yarn.lock` after adding `@material-ui/lab`? (Mine updated after running `yarn install`)

Reply:
I can commit the package-lock as well

Reply:
We shouldn't have both. I would only update yarn.lock, and remove the package-lock file since we're using yarn for this project.